src/cpp/ext/gsm/metadata_exchange.h

```diff
@@ -58,6 +54,11 @@ class ServiceMeshLabelsInjector : public LabelsInjector {
   grpc_core::Slice serialized_labels_to_send_;
 };
 
+// Returns the mesh ID by reading and parsing the bootstrap file. Returns
+// "unknown" if for some reason, mesh ID could not be figured out.
+// Exposed in header for test purposes.
+std::string GetMeshId();
```

Comment:
Please add a comment saying that this is exposed for testing purposes only.

Reply:
Already had it. Made it in caps now :)

Reply:
Whoops, sorry -- I swear I can read. :)